requirements.in

```diff
@@ -10,6 +10,7 @@ python-networkmanager
 PyYAML
 resumable-urlretrieve
 Unidecode
+bleach
```

Comment:
You forgot to add bleach to the `requirements.txt` file.

(also, let's try to keep this file alphabetically ordered? 😛)